src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <section class="calendar calendar--monthLength-30">
+      <div class="calendar__day calendar__day--1 calendar__day--startDay-tue"></div>
```

Comment:
Then here and in all of the divs below it we don't need information about which day it represents, it will simplify things a bit
```suggestion
      <div class="calendar__day day"></div>
```